mapbox/dr-ui
# [Info arch] Fix unwanted nav accordion scroll bars in IE

During info arch testing, we learned that pages with `<Tag>`s in their nav accordions would get horizontal sidebars on IE. Inspired by [this GitHub comment](https://github.com/philipwalton/flexbugs/issues/170#issuecomment-345600356), this PR adds `overflow:hidden` to any parent `<li>` with a `<Tag>` descendant.

| before | after |
| -----| ------|
| <img width="270" alt="image" src="https://user-images.githubusercontent.com/2365503/99723271-94272400-2a77-11eb-9c09-db67618c5e93.png"> | <img width="260" alt="image" src="https://user-images.githubusercontent.com/2365503/99723137-64781c00-2a77-11eb-98c6-6e1d31a65efd.png"> |


## How to test

Start the catalog site to confirm the presence of the unwanted scrollbar in IE. In one of the catalog site's guides, add `tag: fundamentals` to the frontmatter and refresh to page to confirm scrollbars are gone.

## QA checklist




Open the test cases app locally on:
